apps/teams-test-app/e2e-test-data/meeting.json

```diff
@@ -203,6 +203,18 @@
       "expectedAlertValue": "requestAppAudioHandling called",
       "expectedTestAppValue": "requestAppAudioHandling() succeeded: isHostAudioless=true"
     },
+    {
+      "title": "requestAppAudioHandling API Call - register audioDeviceSelectionChanged Handler",
```

Comment:
Suggest name by following pattern: "requestAppAudioHandling API Call - register audioDeviceSelectionChanged Handle **- Success**"

(We would like to know for a case that whether we expect success or error)

Reply:
Also I think you need to update Web Host SDK part first so that you could find this event name in side panel.

Reply:
@KangxuanYe , I notice e2e pipeline fails. So I need to merge the PR in meta-hub-sdk first, then this?

Reply:
> Suggest name by following pattern: "requestAppAudioHandling API Call - register audioDeviceSelectionChanged Handle **- Success**"
> 
> (We would like to know for a case that whether we expect success or error)

Updated

Reply:
> @KangxuanYe , I notice e2e pipeline fails. So I need to merge the PR in metaos-hub-sdk first, then this?

Correct. Because E2E test is testing against on main branch of that repo. And I believe you have another PR on updating test app to find the corresponding event name. You need to finish that part and get it checked in and then come to this place.

Reply:
Got it, pushing that PR now. Will get back to this once that is merged.